repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -118,14 +118,12 @@
         </ul>
     </div>
 
-    <#if publication.administrativeSources?has_content >
-    <div data-uipath="ps.publication.administrative-sources">
-        <h4>Administrative sources:</h4>
-        <#if document.administrativeSources?has_content >
+    <#if publication.administrativeSources?has_content>
+    <div class="content-section">
+        <h4>Administrative Sources</h4>
```

Comment:
Do you still want to display the Administrative Sources title as a placeholder if you have no content?

Reply:
Karen says it's fine as it is.